tests/nightly/jenkins-nightly-run.sh

```diff
@@ -16,84 +16,75 @@
 ESX_60_VERSION="ob-5251623"
 VC_60_VERSION="ob-5112509"
 
+ESX_65_VERSION="ob-7867845"
+VC_65_VERSION="ob-7867539"
+
 ESX_67_VERSION="ob-8169922"
 VC_67_VERSION="ob-8217866"
 
+DEFAULT_TESTCASES="tests/manual-test-cases/Group5-Functional-Tests tests/manual-test-cases/Group13-vMotion tests/manual-test-cases/Group21-Registries tests/manual-test-cases/Group23-Future-Tests"
```

Comment:
This was just moved from below - need to figure out if:
a. it's actually the full set of tests we should be running
b. why we don't have a _nimbus-test-cases_ directory or similar for those expected to be run automatically